rdtools/analysis_chains.py

```diff
@@ -142,6 +143,7 @@ def __init__(self, pv, poa_global=None, temperature_cell=None, temperature_ambie
             'ad_hoc_filter': None  # use this to include an explict filter
         }
         self.filter_params_aggregated = {
+            'hampel_filter': {},
```

Comment:
It's possible that this will be part of the 3.0 release. The main goal here is to have a branch we can test and validate. Once we've done the validation exercise with this and other filtering options that have been proposed, we'll have a better sense for what the ultimate merge/version strategy will be. So at this point, I think it could go either way. The main thing is what makes it easiest for @kanderso-nrel to get this in the Fleets pipeline for validation and testing.

Reply:
I would argue for separate example notebooks here.  My assumption is that with a major RdTools release (3.0?) we'll be making some of these improvements part of the new 'default' analysis.  In this case, a comparison with the old and new filter methods will mainly be a legacy question that someone could check out in a side notebook if they're really interested.  The Hampel filter approach is very lightweight and would be easy to enable by default.  The clearsky filtering requires additional processing and availability of csi irradiance, so might be better as an option.  I can re-work the example notebook to only focus on the hampel / clearsky filter improvements.